google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/resources/ProjectSelectorItem.java

```diff
@@ -30,13 +30,20 @@
   private static final Color TEXT_NON_SELECTION_COLOR = UI.getColor("");
   private static final Color HOVER_COLOR = UI.getColor("link.foreground");
 
-  ProjectSelectorItem(boolean selected, boolean hovered) {
+  ProjectSelectorItem() {
     setBorder(BorderFactory.createEmptyBorder(2, 15, 2, 0));
     setOpaque(false);
     setHorizontalAlignment(SwingConstants.LEFT);
     setVerticalAlignment(SwingConstants.CENTER);
     setBackground(BACKGROUND_NON_SELECTION_COLOR);
+    // default is non-selected, non-hover state.
```

Comment:
nit: I don't think this comment is needed

Reply:
ok, agree.